docs/data/guides/telemetry/telemetry.md

```diff
@@ -1,24 +1,63 @@
 ---
+title: MUI X Telemetry guide
 packageName: '@mui/x-telemetry'
 ---
 
-# MUI X Telemetry guide
+# MUI X Telemetry guide [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
 
 <p class="description">MUI X Telemetry collects anonymous usage data to help improve the library. This guide walk you through how to opt-in, opt-out, and configure telemetry.</p>
 
-## Opting In
+## Why is Telemetry collected?
 
-Currently, **Telemetry is disabled by default**. To opt-in, you can use one of the following methods:
+Developers widely use MUI X to build highly performant and customized applications.
+To further elevate the product experience, and design better features, we aim to gain deeper insights from our users.
 
-### Setting the Environment Variable
+So far, we've only been relying on manual ways such as surveys and community engagement to gather feedback. Although valuable, these are inherently limited in reach, capturing only a fraction of our user base.
+
+Telemetry gives us a clearer picture of how MUI X is used in the real world—what features are most helpful, what needs work, and where we should focus next. While community feedback and surveys are valuable, they only tell a part of the story. Telemetry fills in the gaps and helps us make smarter, faster product decisions.
+
+By opting in, you directly contribute to shaping the future of MUI X. You help us build better tools for you and thousands of other developers—without any impact on your production app. And of course, participation is entirely optional and telemetry is off by default.
+
+## What is being collected?
+
+We track the following details anonymously during development mode:
+
+- Unique session ID: This is generated on each run with UUID.
+- General machine information (operating system, version and whether or not the command was run within CI)
```

Comment:
We don't collect operating system, version not really, we hash it.

```suggestion
- A hash of general machine information (user agent, machine id, etc).
```

I have created https://github.com/mui/mui-x/pull/17282/files to try to make sense of what we use.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/User-Agent